src/common/imageio.c

```diff
@@ -531,6 +541,11 @@ int dt_imageio_is_hdr(const char *filename)
 #endif
 #ifdef HAVE_LIBAVIF
        || !strcasecmp(c, ".avif")
+#endif
+#ifdef HAVE_LIBHEIF
+       || !strcasecmp(c, ".heif")
```

Comment:
Same as above, could leverage libheif for ".avif" read if HAVE_LIBAVIF is undefined